Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -1388,6 +1388,74 @@ end Induction
 
 end IntermediateField
 
+section Minpoly
+
+open AlgEquiv
+
+variable {K L : Type _} [Field K] [Field L] [Algebra K L]
+namespace AdjoinRoot
+
+/-- The canonical algebraic equivalence between `AdjoinRoot p` and `AdjoinRoot q`, where
+  the two polynomial `p q : K[X]` are equal.-/
+noncomputable def id_algEquiv {p q : K[X]} (hp : p ≠ 0) (h_eq : p = q) :
+    AdjoinRoot p ≃ₐ[K] AdjoinRoot q :=
```

Comment:
Maybe this should be called `algEquivOfEq`? And the two theorems `id_algEquiv_def'` and `id_algEquiv_def` should be called `coe_algEquivOfEq` and `algEquivOfEq_toAlgHom`?

Reply:
Should it be `algEquivOfEq` or `algEquiv_of_eq`? The second option looks a bit better to me, but I see both patterns in Mathlib.

Reply:
https://leanprover-community.github.io/contribute/naming.html

> All other terms of Types (basically anything else) are in `lowerCamelCase`.

According to naming conventions, it should be called `algEquivOfEq`, since it's an `AlgEquiv` which is a `Type`.

Reply:
Done!